Makefile

```
@@ -89,10 +89,29 @@ gcp_k8s: init  ## Deploys GKE K8s cluster (CPs only)
 		done; \
 		'
 
+.PHONY: k8s_auth
+k8s_auth: k8s_auth_gcp k8s_auth_aws k8s_auth_azure  ## Refreshes k8s auth token
+k8s_auth_%:
+	@echo "Refreshing k8s_auth..."
+	@/bin/sh -c '\
+		set -e; \
+		index=0; \
+		jq -r '.$*_k8s_regions[]' terraform.tfvars.json | while read -r region; do \
+		echo "cloud=$* region=$$region cluster_id=$$index"; \
+		cd "infra/$*/k8s_auth"; \
+		terraform workspace new $*-$$index-$$region || true; \
+		terraform workspace select $*-$$index-$$region; \
+		terraform init; \
+		terraform apply -refresh=false ${terraform_apply_args} -var-file="../../../terraform.tfvars.json" -var=$*_k8s_region=$$region -var=cluster_id=$$index; \
+		terraform workspace select default; \
+		cd "../../.."; \
```

Index doesn't increment - multi-regions under same cloud wont have auth state available.
```suggestion
		index=$$((index+1)); \
		cd "../../.."; \
```